.travis.yml

```diff
@@ -5,8 +5,8 @@ language: go
 go_import_path: github.com/coreos/ignition
 
 go:
-  - 1.8
-  - 1.9
+  - "1.8"
+  - "1.11"
```

Comment:
Is it worth continuing to test the two latest go versions in addition to an older version (i.e. `1.8`, `1.10`, & `1.11` in this case)?

Reply:
Eh, sure. travis is "free".